.github/workflows/dance.yml

```diff
@@ -97,14 +99,14 @@ jobs:
       - name: Start environment
         run: bin/task env-up-detach
         env:
-          FERRETDB_IMAGE: ${{ inputs.ferretdb_image || 'ghcr.io/ferretdb/ferretdb:1' }}
-          POSTGRES_IMAGE: ${{ inputs.postgres_image || 'postgres:16.4' }}
+          FERRETDB_IMAGE: ${{ inputs.ferretdb_image || 'ghcr.io/ferretdb/ferretdb:2.0.0' }}
+          POSTGRES_IMAGE: ${{ inputs.postgres_image || 'ghcr.io/ferretdb/postgres-documentdb:17-0.102.0-ferretdb-2.0.0' }}
```

Comment:
Let's remove those default values – having them in the Docker Compose configuration is enough.

Reply:
Removed them.